src/app/item-popup/Plug.tsx

```diff
@@ -130,6 +131,12 @@ export function PerkCircleWithTooltip({
       craftingData={socketInfo.craftingData?.[plug.plugDef.hash]}
     />
   );
+
+  const isRecommendedPerk =
+    wishlistRoll &&
+    (wishlistRoll.wishListPerks.has(plug.plugDef.hash) ||
+      wishlistRoll.wishListPerks.has(enhancedToPerk[plug.plugDef.hash]));
```

Comment:
This is just `isWishListPlug` below?

Reply:
slightly different typings but i could make the double .has a shared function yeah

Reply:
should be good